README.md

```diff
@@ -211,6 +211,49 @@ following advantages over standard Web3 subscriptions:
   with a lower likelihood, for outgoing requests to be lost, so you should still
   have error handling as with any network request.
 
+## Alchemy's Transfers API
```

Comment:
We might consider putting this in the "Alchemy's Enhanced APIs" section below it, unless we're intentionally branding this as separate from the Enhanced APIs.